packages/hardhat-plugin/src/index.ts

```diff
@@ -235,7 +245,8 @@ task("deploy2")
         parameters = resolveParametersString(parametersInput);
       }
 
-      const isHardhatNetwork = hre.network.name === "hardhat";
+      // TODO: bring back check with file journaling proper
+      const isHardhatNetwork = true; // hre.network.name === "hardhat";
```

Comment:
This check should be done with a request to`web3_clientVersion`

Reply:
Cool. I will do that change in the store to file/deployment loader PR as that is where the check has moved.

Reply:
After discussion we opted to leave the check. We want ignition in tests to be ephemeral but ignition against a localhost node to record a file deployment.